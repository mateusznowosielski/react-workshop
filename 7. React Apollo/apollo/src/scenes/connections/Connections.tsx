import * as React from 'react';

const Connections: React.FunctionComponent<any> = () => {
  return (
    <div>
      <h1>Connections</h1>
    </div>
  )
};

export default Connections;