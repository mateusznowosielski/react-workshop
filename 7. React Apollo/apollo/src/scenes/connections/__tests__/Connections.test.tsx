import * as React from 'react';
import { cleanup, render } from 'react-testing-library';
import Connections from 'scenes/connections/Connections';

afterEach(cleanup);

it('verifying Connections to be rendered', () => {
  const { container } = render(<Connections/>);

  expect(container).toMatchSnapshot();
});