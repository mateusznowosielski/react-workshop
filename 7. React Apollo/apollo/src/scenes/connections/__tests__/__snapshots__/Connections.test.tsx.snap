// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`verifying Connections to be rendered 1`] = `
<div>
  <div>
    <h1>
      Connections
    </h1>
  </div>
</div>
`;
